bridge/client/app/_components/ktb-secrets-list/ktb-secrets-list.component.ts

```diff
@@ -15,7 +15,7 @@ export class KtbSecretsListComponent implements OnInit, OnDestroy {
   private readonly unsubscribe$ = new Subject<void>();
   private closeConfirmationDialogTimeout?: ReturnType<typeof setTimeout>;
 
-  public tableEntries: DtTableDataSource<object> = new DtTableDataSource();
+  public tableEntries: DtTableDataSource<Secret> = new DtTableDataSource();
   public currentSecret?: Secret;
 
   public deleteSecretDialogState: string | null = null;
```

Comment:
Maybe change it to `public deleteSecretDialogStage?: string` to have a default value of `undefined`, else we would have two different styles on how to deal with default/null values in the project. Do we stick to `undefined` or `null` as default value?